compiler/rustc_trait_selection/src/traits/error_reporting/suggestions.rs

```diff
@@ -1351,13 +1352,34 @@ impl<'tcx> TypeErrCtxtExt<'tcx> for TypeErrCtxt<'_, 'tcx> {
                         );
```

Comment:
Should the parentheses be added to this suggestion too?

Reply:
I don't think so, I go through all the UI testcase come from this part,
they are all with single argument, and scenrio here is both `&` and `&mut` can be added for borrowing.

I haven't figure out an case which we should add parentheses here.